example/broker.cpp

```diff
@@ -66,6 +64,21 @@ class server_no_tls {
 
 #if defined(MQTT_USE_TLS)
 
+bool verify_certificate(bool preverified, boost::asio::ssl::verify_context& ctx, std::shared_ptr<MQTT_NS::optional<std::string>> const& username) {
+    if (!preverified) return true;
+    int depth = X509_STORE_CTX_get_error_depth(ctx.native_handle());
+    if (depth > 0) return true;
```

Comment:
what about this depth check, should return false ?

Reply:
I think that depth parameter should be passed as broker option.